src/libutil/linux/cgroup.cc

```diff
@@ -144,4 +145,35 @@ CgroupStats destroyCgroup(const Path & cgroup)
     return destroyCgroup(cgroup, true);
 }
 
+std::string getCurrentCgroup()
```

Comment:
Would be nice to use this function in `getMaxCPU()` which also looks up the current cgroup.

Reply:
Done